src/components/fieldset/template.test.js

```diff
@@ -61,6 +61,15 @@ describe('fieldset', () => {
     expect($legend.text().trim()).toEqual('What is your address?')
   })
 
+  it('allows you to set the initial aria-describedby attribute', () => {
+    const $ = render('fieldset', {
+      describedBy: 'parent-fieldset-hint-id'
+    })
+
+    const $component = $('.govuk-fieldset')
+    expect($component.attr('aria-describedby')).toEqual('parent-fieldset-hint-id')
```

Comment:
Very much non-blocking, but as `$component` only gets used once and there's nothing complicated going on I'd probably just chain it directly

```suggestion
    expect($('.govuk-fieldset').attr('aria-describedby')).toEqual('parent-fieldset-hint-id')
```

Reply:
It's like this in lots of tests, so I copied this pattern. Shall we leave it?

Reply:
Going to leave this as it matches other assertions where `$component` is only used once.

See:

https://github.com/alphagov/govuk-frontend/blob/master/src/components/fieldset/template.test.js#L28

https://github.com/alphagov/govuk-frontend/blob/master/src/components/fieldset/template.test.js#L39

https://github.com/alphagov/govuk-frontend/blob/master/src/components/fieldset/template.test.js#L50

etc